### PR TITLE
User stories dg

### PR DESCRIPTION
**Pull request by prit3010**
Update user stories and batch-add UML diagram

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -339,32 +339,53 @@ These are possible things to work on for future iterations.
...
+UML Diagram:
+![BatchAdd](images/diagrams/BatchAddUML.png)
 
-Step 2: The `parse` function returns a `BatchAddCommand` which is then executed. In this `execute` function, the first
-step would be to read the information in the CSV file (`getInfo` function). A `BufferedReader` object is used to read the CSV file and write it
-into a `List<AddCommand>`. If file does not exist in the folder, a `FileNotFound` exception is thrown too.
+**Steps:**
 
-Step 3. Once `getInfo` returns a `List<AddCommand>`, the list will then be iterated through to execute each `AddCommand`
+**Step 1**: User launches tha application.
```
Comment (Kok-je) [inline docs/DeveloperGuide.md:350]: tha? I think you meant to spell "the"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -339,32 +339,53 @@ These are possible things to work on for future iterations.
...
+Note: If no argument is provided or file not in CSV format, a`ParseException` will be thrown.
+
+**Step 3**: The `parse` function returns a `BatchAddCommand` which is then executed. In this `execute` function, the first
+step would be to read the information in the CSV file (`getInfo` function). A `BufferedReader` object is used to read the CSV file and converts 
+each row to a string of arguments (following the add command requirements) and creates a new `AddCommand`. These new `AddCommands` will be added 
+into a `List<AddCommand>`. 
+
+Note: If file does not exist in the folder, a `FileNotFound` exception is thrown too.
+
+**Step 4**: Once `getInfo` returns a `List<AddCommand>`, the list will then be iterated through to execute each `AddCommand`
```
Comment (Kok-je) [inline docs/DeveloperGuide.md:365]: Fullstop consistency in this line

**Review by Kok-je — CHANGES_REQUESTED** [review]
Some minor errors. Please make necessary edits

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (kevinchangjk) [reply docs/DeveloperGuide.md:350]: Nah it's style.

Comment (prit3010) [reply docs/DeveloperGuide.md:350]: yes

**Review by Kok-je — APPROVED** [review]
LGTM

**Pull request merged**